annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/helpers/FastPackageNameDecoder.java

```diff
@@ -0,0 +1,7 @@
+package com.exactpro.epfast.annotation.processing.helpers;
+
+public class FastPackageNameDecoder {
+    public static String decode(String refactoredPath) {
```

Comment:
Don't decode it back. Just store original value. If you need to store several values in map key or value, create new class. It will benefit code readability as well